spec/latest/schema.org/index.html

```diff
@@ -446,16 +446,16 @@
           <td>hydra:required</td>
         </tr>
         <tr>
-          <td>readonly</td>
+          <td>readable</td>
```

Comment:
As the note at the beginning of this documents states, this is document "serves purely as a historical reference". As such, we don't update it anymore. Please rever the changes to this file.